src/libos/src/unit_test/README.md

```diff
@@ -0,0 +1,31 @@
+# Occlum Unit Testing
+
+## Compiling the tests
+
+```OCCLUM_UT``` controls whether ```make``` compiles the unit tests into the
+occlum library. To compile the tests, run
+
+```
+$ OCCLUM_UT=1 make
+```
+
+Crate inventory does not support debug mode, so current unit test framework
+only supports release build.
+
+## Running the tests
+The unit tests are invoked through `unit_test` in `test`. Each test runs in a
+separate process. The tests can be run in different ways:
+```
+# Run unit_test along with other tests:
+$ make test OCCLUM_UT=1
+
+# Only run unit_test
+$ make test TESTS=unit_test
+
+# Assign the test_name, like net::test::test_iov, to BIN_ARGS to run specific test
+$ make test BIN_ARGS=<test_name> TESTS=unit_test
+```
```

Comment:
I think the doc should be put to where it would be most likely to be used.

For example, when running unit tests, a developer would most likely to try to find relevant info in the `test` directory, rather than the `src` directory. Actually, mentioning `make test` and its usage in source code itself is not a good idea, which entangles the source code and the test infrastructure.

Reply:
Developers can find all they want to know about occlum unit test in one file. I can put the contents to different places if you have any concrete idea.